packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -118,8 +118,6 @@ class ImageConfiguration {
     result.write('ImageConfiguration(');
     bool hasArguments = false;
     if (bundle != null) {
-      if (hasArguments)
-        result.write(', ');
```

Comment:
that's a pity, it'll make copy-and-paste errors much more likely. An //ignore isn't any better though I guess since it would also make the first one different than the rest...

I guess the real solution is to make this Diagnosticable, but that's out of scope for this PR (unless you're looking for bonus points).

Reply:
@Hixie What am I missing?  As I read it, `hasArguments` is always false at line 121, so the extra ", " was never printed, it's just dead code.

Where does the copy-and-paste error come in?